### PR TITLE
[Feature] Fixed validators bugs.

### PR DESCRIPTION
**Pull request by StanleyOned**
Fixed bugs for only specifying minimum characters or passing an empty array for Validation Type.

### CODE REVIEW

File: ios/DevKit/DevKit.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
```
Comment (jgsamudio) [inline ios/DevKit/DevKit.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Do you know why this was added to the project? Can we remove it?

File: ios/DevKit/DevKit/Classes/Validation/FormValidator.swift [hunk 2]
```diff
@@ -35,6 +35,7 @@ public class FormValidator {
     ///   - validators: The validators for your password.
     public func isValid(_ password: String, validators: [ValidationType]) -> Bool {
         return isValid(password, passwordRegex: ValidationType.getRegexCode(with: validators))
+               || ValidationType.checkForOnlyMinCharacters(password, validators: validators)
```
Comment (jgsamudio) [inline ios/DevKit/DevKit/Classes/Validation/FormValidator.swift]: We should update our tests to cover for this case

File: ios/DevKit/DevKit/Classes/Validation/ValidationType.swift [hunk 3]
```diff
@@ -63,6 +63,22 @@ public enum ValidationType {
         return regexCode
     }
     
+    static func checkForOnlyMinCharacters(_ password: String, validators: [ValidationType]) -> Bool {
```
Comment (jgsamudio) [inline ios/DevKit/DevKit/Classes/Validation/ValidationType.swift]: We should add unit tests for the logic in this function.

File: ios/DevKit/DevKit.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 1 above)
Comment (StanleyOned) [reply ios/DevKit/DevKit.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Not completely sure where this file/changes came from.

File: ios/DevKit/DevKit/Classes/Validation/FormValidator.swift (hunk 2 above)
Comment (StanleyOned) [reply ios/DevKit/DevKit/Classes/Validation/FormValidator.swift]: will do 👍

**Review by jgsamudio — APPROVED** [review]

**Pull request merged**